src/StringTools/InternableString.cs

```diff
@@ -304,28 +304,60 @@ public override string ToString()
         /// <returns>A stable hashcode of the string represented by this instance.</returns>
         public override unsafe int GetHashCode()
```

Comment:
Is this still djb2?

Reply:
I will fix to say something like "djb2-like" and link to the dotnet/runtime method.

Reply:
I have updated the comment.